### PR TITLE
[IN-115][Reviews] Add preprint doi pending minting message

### PR DESCRIPTION
**Pull request by adlius**
## Purpose

Fix the problem that preprint detail page for moderation shows the link for DOI before it is minted.

## Summary of Changes

Add conditionals in `preprint-detail.hbs` to watch for `preprintDoiCreated`.

## Side Effects / Testing Notes

None.

## Ticket

https://openscience.atlassian.net/browse/IN-115

### CODE REVIEW

File: app/templates/preprints/provider/preprint-detail.hbs [hunk 1]
```diff
@@ -90,11 +90,15 @@
...
-                                <a href={{preprint.preprintDoiUrl}}>{{extract-doi preprint.preprintDoiUrl}}</a>
-                            {{else if (not preprint.isPublished)}}
-                                <h4 class="p-v-md f-w-md"><strong>{{t "content.preprintDOI"}}</strong></h4>
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprintDOI"}}</strong></h4>
+                            {{#if preprint.isPublished}}
+                                {{#if preprintDoiCreated}}
+                                    <a href={{preprint.preprintDoiUrl}}>{{extract-doi preprint.preprintDoiUrl}}</a>
+                                {{else}}
+                                    <p> {{extract-doi preprint.preprintDoiUrl}} </p>
+                                    <p> {{t "content.preprintPendingDOIMinted"}} </p>
```
Comment (baylee-d) [inline app/templates/preprints/provider/preprint-detail.hbs]: You probably don't need spaces between the `<p>` and `{{` on lines 98 or 99 here

File: app/templates/preprints/provider/preprint-detail.hbs [hunk 2]
```diff
@@ -90,11 +90,15 @@
...
 
                         <section class="p-t-xs">
-                            {{#if preprint.preprintDoiUrl}}
-                                <h4 class="p-v-md f-w-md"><strong>{{t "content.preprintDOI"}}</strong></h4>
-                                <a href={{preprint.preprintDoiUrl}}>{{extract-doi preprint.preprintDoiUrl}}</a>
-                            {{else if (not preprint.isPublished)}}
-                                <h4 class="p-v-md f-w-md"><strong>{{t "content.preprintDOI"}}</strong></h4>
+                            <h4 class="p-v-md f-w-md"><strong>{{t "content.preprintDOI"}}</strong></h4>
+                            {{#if preprint.isPublished}}
+                                {{#if preprintDoiCreated}}
```
Comment (baylee-d) [inline app/templates/preprints/provider/preprint-detail.hbs]: Where is `preprintDoiCreated` being declared?  Or is it an attribute in the model?

**Review by baylee-d — CHANGES_REQUESTED** [review]
Just some small formatting changes and a question on one of the variables.  Other than that, it looks good.

**Review by alexschiller — APPROVED** [review]
LGTM 🔍 🚐

**Review by alexschiller — CHANGES_REQUESTED** [review]
I changed the branch to next-interfaces. Can you add the CHANGELOG please?

**Review by alexschiller — APPROVED** [review]

**Pull request merged**